src/views/Login/Login.vue

```diff
@@ -172,6 +172,9 @@ export default {
       this.disableSubmitButton = true;
       const username = this.userInfo.username;
       const password = this.userInfo.password;
+      if (username === null && password === null) {
```

Comment:
When exactly is this displayed?

Reply:
After changing the hostname, the BMC GUI became logged out, then I tried logging back in using my service credentials, but I couldn't. It worked after I refreshed the BMC GUI webpage and logged in with my service credentials.

Reply:
This is added to display this error message "If you are failed to login, refresh the browser." If we remove this message from login page, this part will also be removed.